app/src/main/index.ts

```diff
@@ -28,7 +28,7 @@ function createWindow(): void {
   mainWindow.on("ready-to-show", () => {
     mainWindow.show();
     // Default open DevTools in development
-    if (is.dev) {
+    if (is.dev && process.env["NODE_ENV"] != "production") {
```

Comment:
The different semantics of `is.dev` (here; checks against app.isPackaged) and `isDev` (in `electron.vite.config.ts`) are a bit confusing - maybe worth a comment.